### PR TITLE
vm: add PUSHT and PUSHF opcodes

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Port https://github.com/neo-project/neo-vm/pull/497.

Draft until the reference PR merge.

### CODE REVIEW

**Review by fyrchik — APPROVED** [review]

File: pkg/core/fee/opcode.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ var coefficients = [256]uint16{
 	opcode.PUSHINT64:    1 << 0,
 	opcode.PUSHINT128:   1 << 2,
 	opcode.PUSHINT256:   1 << 2,
+	opcode.PUSHT:        1 << 0,
+	opcode.PUSHF:        1 << 0,
```
Comment (roman-khimov) [inline pkg/core/fee/opcode.go:24]: To be checked again after the respective core change.

**Review by roman-khimov — COMMENTED** [review]
It's OK, but we'll merge it for 3.5.0-compatible release.

File: pkg/core/fee/opcode.go (hunk 1 above)
Comment (AnnaShaleva) [reply pkg/core/fee/opcode.go:24]: They are the same, see https://github.com/neo-project/neo/pull/2832/files/66d5cb1a36a2d3b9c2b9496a363a694aa18dae38..fa6ab8f33076dc84448723365bef55b71503a4f9

**Comment by AnnaShaleva** [comment]
Updated.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**